Sprint-2/interpret/invert.js

```diff
@@ -10,20 +10,21 @@ function invert(obj) {
   const invertedObj = {};
 
   for (const [key, value] of Object.entries(obj)) {
-    invertedObj.key = value;
+    invertedObj[value] = key;
   }
 
   return invertedObj;
 }
 
-// a) What is the current return value when invert is called with { a : 1 }
+module.exports = invert;
+// a) What is the current return value when invert is called with { a : 1 } - current return value is {key: 1}
 
-// b) What is the current return value when invert is called with { a: 1, b: 2 }
+// b) What is the current return value when invert is called with { a: 1, b: 2 } - current return value is {key: 2}
 
-// c) What is the target return value when invert is called with {a : 1, b: 2}
+// c) What is the target return value when invert is called with {a : 1, b: 2} -  current returns an array of values
```

Comment:
target value == expected value returned by the correctly implemented function when the parameter is  `{a : 1, b: 2}`